packages/editor/src/components/post-panel-row/style.scss

```diff
@@ -29,6 +29,8 @@
 		text-wrap: pretty;
 		height: auto;
 		min-height: $button-size-compact;
+		display: inline-block;
```

Comment:
Why do we need `display: inline-block;`?

Reply:
Apologies, My browser was not responding to this change, that why I added it. Will remove it, as it is not necessary.